modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -228,9 +228,8 @@ export default class Checkbox extends React.Component<CheckboxProps> {
   public static defaultProps = {
     checked: false,
     label: '',
-    id: uuid(),
   };
-
+  private id = uuid();
```

Comment:
Nit: can you put new lines on either side of this? We generally put a space between all class members/functions